src/content/docs/apis/nerdgraph/examples/nerdgraph-api-secret-management-service.mdx

```diff
@@ -0,0 +1,255 @@
+---
+title: "NerdGraph tutorial: Secret Management Service"
+metaDescription: "Learn how to use the Secret Management Service to securely store and manage your secrets."
+freshnessValidatedDate: never
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+The Secret Management Service is a secure and scalable way to store and manage your secrets. It provides a centralized location for storing and managing secrets, such as API keys to reduce fragmentation and complexity across different services in New Relic. 
+You can store any string value as a secret, and call them programatically during runtime.
+
+Note that there's no data boundary for storing secrets; all secrets reside in US in a global DB enclave.
```

Comment:
I think it should be in the US. 

Double-check.

Semicolons make sentences unnecessarily complex. Let's split the note into two sentences.

And should we use the callout component instead? This seems like an important note.